modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp

```diff
@@ -232,6 +232,7 @@ renderResponse.setTitle(headerTitle);
 
 			<liferay-ui:error exception="<%= FileNameException.class %>" message="please-enter-a-file-with-a-valid-file-name" />
 			<liferay-ui:error exception="<%= FileNameExtensionException.class %>" message="the-file-name-cannot-be-empty-or-without-extension" />
+			<liferay-ui:error exception="<%= FileNameExtensionMismatchException.class %>" message="the-file-extension-cannot-be-different-from-filename-extension" />
```

Comment:
The key has to match the value. You're missing a "the" in the key (..-different-from-*the*-filename-...). Also (but only in the key), "filename" must be split (i.e. "file-name"); leave it as "filename" in the value.

Reply:
In case of doubt, check current usages in Language.properties (grep -F 'file-name' Language.properties).

Reply:
Ok, thank you for the tip.